assets/js/app/DataLayers/genes.js

```diff
@@ -34,18 +41,40 @@ LocusZoom.DataLayers.add("genes", function(layout){
             + this.layout.track_vertical_spacing;
     };
 
-    // A gene may have arbitrarily many transcripts, but this data layer isn't set up to render them yet.
-    // Stash a transcript_idx to point to the first transcript and use that for all transcript refs.
+    /**
+     * A gene may have arbitrarily many transcripts, but this data layer isn't set up to render them yet.
+     * Stash a transcript_idx to point to the first transcript and use that for all transcript refs.
+     * @member {number}
+     * @type {number}
+     */
     this.transcript_idx = 0;
-    
+
+    /**
+     * TODO: Description. How does this differ from similarly named properties?
```

Comment:
`this.tracks` is nothing more than a number that stores the number of tracks in the data layer. Each time a new region is requested an arbitrary number of genes will come back with arbitrary overlaps, so `assignTracks()` runs. It resets `this.tracks` to 1 and cycles through, positioning genes on different tracks to prevent overlap while (ideally) keeping tracks to a minimum.

Tracks on a genes panel are not special or indicative of any meaning, they're just rows used to vertically separate overlapping genes.